physics/barycentric_rotating_dynamic_frame_body.hpp

```diff
@@ -173,6 +96,54 @@ BarycentricRotatingDynamicFrame<InertialFrame, ThisFrame>::ReadFromMessage(
 }
 
 template<typename InertialFrame, typename ThisFrame>
+Vector<Acceleration, InertialFrame>
+BarycentricRotatingDynamicFrame<InertialFrame, ThisFrame>::
+    GravitationalAcceleration(Instant const& t,
+                              Position<InertialFrame> const& q) const {
+  return ephemeris_->ComputeGravitationalAccelerationOnMasslessBody(q, t);
+}
+
+template<typename InertialFrame, typename ThisFrame>
+SecondOrderRigidMotion<InertialFrame, ThisFrame>
+BarycentricRotatingDynamicFrame<InertialFrame, ThisFrame>::Motion(
+    Instant const& t) const {
+  DegreesOfFreedom<InertialFrame> const primary_degrees_of_freedom =
+      primary_trajectory_->EvaluateDegreesOfFreedom(t, &primary_hint_);
+  DegreesOfFreedom<InertialFrame> const secondary_degrees_of_freedom =
+      secondary_trajectory_->EvaluateDegreesOfFreedom(t, &secondary_hint_);
+
+  Vector<Acceleration, InertialFrame> const primary_acceleration =
+      ephemeris_->ComputeGravitationalAccelerationOnMassiveBody(primary_, t);
+  Vector<Acceleration, InertialFrame> const secondary_acceleration =
+      ephemeris_->ComputeGravitationalAccelerationOnMassiveBody(secondary_, t);
+
+  auto const to_this_frame = ToThisFrameAtTime(t);
+
+  // TODO(egg): TeX and reference.
+  RelativeDegreesOfFreedom<InertialFrame> const primary_secondary =
+      primary_degrees_of_freedom - secondary_degrees_of_freedom;
+  Variation<AngularVelocity<InertialFrame>> const
+      angular_acceleration_of_to_frame =
+          (Wedge(primary_secondary.displacement(),
+                 (primary_acceleration - secondary_acceleration)) * Radian -
+           2 * to_this_frame.angular_velocity_of_to_frame() *
```

Comment:
Please keep the variable `Ω_inertial'.  This is messy enough without making the formula even messier.